submissions/VolodymyrRutskyi/task_js_dom/main.js

```diff
@@ -0,0 +1,186 @@
+let data = [
+    {
+        name: 'Concat',
+        image: 'img/concat.jpg',
+        description: [
+            `<p>Метод concat создаёт новый массив, состоящий
```

Comment:
Avoid tags. Think how it is possible to implement this with the help of a method `Array.map`

Reply:
fixed